### ISSUE TITLE
Carousel: accessibility aria label

### ISSUE THREAD

**Issue by djmj** [primefaces/primefaces#13793]
### Description

In `<p:carousel>` the buttons: prev, next and page indicators are not accessible in PF 14

It can be seen testing the showcase with chrome lighthouse tool:

https://www.primefaces.org/showcase/ui/data/carousel.xhtml

### Describe the solution you would like

aria-labels for:

- prev button `aria-label="Back"`
- next button `aria-label="Next"`
- indicators: `aria-label="{Page: + index}"`
